packages/validation/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@trezor/validation",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
```

Comment:
you need to define `sideEffect: ["src/config.ts"]` beccause there are some side effects happening

Reply:
Could you explain what the side effects are in this case? `yup` should be imported from the package whenever this package is used, so I can't imagine a usecase where anything in the `config` is needed but it's not exported.

Reply:
You are true, I thought it will be safer in case someone will decide to move reexport of yup to index for example.

Let it be as it is now.